spec/helpers/tasklist_helper_spec.rb

```diff
@@ -1,41 +1,32 @@
 require "rails_helper"
 
 RSpec.describe TasklistHelper do
-  let(:now) { Time.utc(2022, 1, 1) }
-
-  around do |example|
-    Timecop.freeze(now) do
-      Singleton.__init__(FormHandler)
-      example.run
-    end
-    Timecop.return
-  end
-
   describe "with lettings" do
     let(:empty_lettings_log) { create(:lettings_log) }
-    let(:lettings_log) { build_stubbed(:lettings_log, :in_progress, needstype: 1, startdate: now) }
+    let(:lettings_log) { build_stubbed(:lettings_log, :in_progress, needstype: 1) }
 
     describe "get next incomplete section" do
       it "returns the first subsection name if it is not completed" do
-        expect(get_next_incomplete_section(lettings_log).id).to eq("household_characteristics")
+        expect(get_next_incomplete_section(lettings_log).id).to eq("property_information")
       end
 
       it "returns the first subsection name if it is partially completed" do
-        lettings_log["tenancycode"] = 123
-        expect(get_next_incomplete_section(lettings_log).id).to eq("household_characteristics")
+        lettings_log["uprn_known"] = 0
+        expect(get_next_incomplete_section(lettings_log).id).to eq("property_information")
       end
     end
 
     describe "get sections count" do
-      let(:real_2021_2022_form) { Form.new("config/forms/2021_2022.json") }
-
-      before do
-        allow(FormHandler.instance).to receive(:get_form).and_return(real_2021_2022_form)
-      end
+      let(:completed_subsection) { instance_double("Subsection", status: :completed, displayed_in_tasklist?: true, applicable_questions: [{ id: "question" }]) }
+      let(:incomplete_subsection) { instance_double("Subsection", status: :not_started, displayed_in_tasklist?: true, applicable_questions: []) }
```

Comment:
I've not seen that you can set the response to methods on an instance_double when defining it. that's nice

Reply:
Yup, if they're short it's quite nice